contents/blog/best-hipaa-compliant-analytics.md

```diff
@@ -0,0 +1,123 @@
+
+You may still need to create a BAA with your cloud hosting provider, as with any self-hosted solution. 
+
+## 4. [Mixpanel](https://mixpanel.com/)
+
+![Mixpanel - hipaa analytics tools](../images/blog/open-source-analytics-tools/mixpanel.png)
+
+Like PostHog, Mixpanel offers a suite of product analytics tools, including funnel and trend analysis. It lacks some specific features, such as Feature Flags, but offers a library of 50 plugins to extend functionality via integrations with other platforms. 
+
+_Unlike_ PostHog, Mixpanel isn’t open-source and can’t be deployed into a users’ own infrastructure. Instead, it must be deployed via the Cloud and data must be shared with Mixpanel directly, meaning [a BAA is required in order to use Mixpanel under HIPAA](https://mixpanel.com/legal/mixpanel-hipaa/). 
+
+### Features & benefits
+
+- All-in-one product analytics suite
+- Group Analytics, for B2B organizations
+- Self-serve, no SQL required
+- SOC 2 certification
+
+### Is Mixpanel HIPAA compliant?
+
+Mixpanel isn’t HIPAA compliant out-of-the-box, but offers a Business Associate Agreement so that it can be used by organizations which need to remain compliant with HIPAA regulations. 
+
+## Is Google Analytics HIPAA compliant?
+
+The short answer is that, no, Google Analytics is not HIPAA compliant. 
+
+According to [Google’s own disclaimers](https://support.google.com/analytics/answer/6366371?hl=en#hipaa&zippy=%2Cin-this-article), Google Analytics cannot be used for handling PHI or ePHI (including IP addresses) by _either_ covered entities or their third-party business associates: 
+
+>Unless otherwise specified in writing by Google, Google does not intend uses of Google Analytics to create obligations under the Health Insurance Portability and Accountability Act, as amended, (“HIPAA”), and makes no representations that Google Analytics satisfies HIPAA requirements. If you are (or become) a Covered Entity or Business Associate under HIPAA, you may not use Google Analytics for any purpose or in any manner involving Protected Health Information unless you have received prior written consent to such use from Google.
+
+Although [Google does offer a BAA for some services](https://support.google.com/a/answer/3407054?hl=en), such as Google Calendar and Google Keep, Google Analytics is not included on this list. In fact, [Google Analytics may even be illegal](https://isgoogleanalyticsillegal.com/) to use in some countries due to the way it collects and stores data. 
+
+> PostHog is an open source analytics tool which enables data and engineering teams to build better products faster and without writing SQL. It can be deployed on your own infrastructure to be used under HIPAA regulations without a Business Associate Agreement.
```

Comment:
```suggestion
> PostHog is an open-source analytics tool which enables data and engineering teams to build better products faster and without writing SQL. It can be deployed on your own infrastructure to be used under HIPAA regulations without a Business Associate Agreement.
```